LondonBritishLibrary/orient/BLorient2083.xml

```diff
@@ -0,0 +1,2474 @@
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="137r" to="177r"/>
+                                    <desc type="Comment">Short notes summarizing the main events of the Passion in the upper margins of the 
+                                        readings for Good Friday.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="218ra" to="218vb"/>
+                                    <desc type="Supplication">Benediction</desc>
+                                    <q xml:lang="gez"><title>ጸሎተ፡ ቡራኬ፡</title> <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1ra"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="de">Gebra Hemamat angekauft fuer 16 Maria Theresia Thaler 
+                                        <date when="1870-08-08">d. 8 August 1870</date>. 
+                                        <persName role="owner" ref="PRS14422FMayer">F. Mayer</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#28va"/>
+                                    <desc>Record with the name of King <persName ref="PRS5616IyasuI">ʾIyāsu I</persName>.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#48va"/>
+                                    <desc type="OwnershipNote">A note in the margin indicating, that this manuscript belonged to the church of
+                                        <placeName ref="LOC7449BabahaAbbo">Babaḥa ʾAbbo</placeName>. A person with the name
+                                        <persName ref="PRS14421WaldaHanna">Walda Ḥannā</persName>, who was possibly a priest of this 
+                                        church, is also mentioned.</desc> 
+                                    <q xml:lang="gez">ዝመጽሐፍ፡ ዘደብረ፡ በበሐ፡ አቦ፡</q>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#122vb"/>
+                                    <desc type="OwnershipNote"><persName role="owner" ref="PRS14419Maryamawit">Māryāmāwit</persName>
+                                        is named as an owner, together with her son <persName ref="PRS14420TaklaHaymanot">Takla Hāymānot</persName>.</desc> 
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#217ra"/>
+                                    <desc>Supplication with the names of Patriarch 
```

Comment:
supplications are usually by the scribe and not additions, this is relevant for dating/origin